### PR TITLE
Adding  Contributing.md

### LINKED ISSUE #10: Add contributing.md

### PR DESCRIPTION
**Pull request by SatyamK23**
Fixes #10

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+## How to contribute to EventFolio
+
+Thanks for considering contribution to Event-Folio. Please discuss the changes you wish to make via issues, email with the owner of this repository before making a change.
```
Comment (TarunISCO) [inline CONTRIBUTING.md]: Suggest to discuss issues in gitter channel.
And as it is our course assignment so taking help from other developers won't be good. However, we can have ideas from everyone for further versions of applications.

Comment (SatyamK23) [reply CONTRIBUTING.md]: All sentences must be added or adding only last sentence in the file. Should I keep the first sentence?

Comment (TarunISCO) [reply CONTRIBUTING.md]: Sorry I can't understand what you said.

Comment (SatyamK23) [reply CONTRIBUTING.md]: Are the rest of the statements in the Contributing.md fine?

Comment (TarunISCO) [reply CONTRIBUTING.md]: I think yes.

Comment (SatyamK23) [reply CONTRIBUTING.md]: All sentences in the review which you have suggested must be included or only the last sentence. Should I include the first sentence Thanks for considering contribution to Event-Folio?

**Pull request merged**